### PR TITLE
Update plotting

### PR DESCRIPTION
**Pull request by wbaizer**
This is the first step towards coherent plotting across questions. Will close issues in subsequent PR when I document better

Fixes #139
Fixes #140
Fixes #74
Fixes #69
Fixes #64
Fixes #112
Fixes #63

### CODE REVIEW

**Review by ghost — COMMENTED** [review]
:white_check_mark: A review job has been created and sent to the PullRequest network.  
*****  
Check the status or cancel PullRequest code review [here](https://will-fe7b.pullrequest.tech/dash/projects/1/wbaizer/ought-copy/reviews).

File: ergo/__init__.py [hunk 1]
```diff
@@ -31,3 +33,5 @@
     to_float,
     uniform,
 )
+
+warnings.filterwarnings(module="plotnine", action="ignore")
```
[the remark this answers is not in the document]
Comment (wbaizer) [reply ergo/__init__.py]: I think your right, perhaps we will just add that in sample notebooks to be more explicit

File: ergo/metaculus.py [hunk 2]
```diff
@@ -367,8 +373,8 @@ def sample_normalized_community(self):
         # how spread out we actually expect the probability mass to be
         outside_range_scale = 0.02
 
-        sample_below_range = -abs(np.random.logistic(0, outside_range_scale))
-        sample_above_range = abs(np.random.logistic(1, outside_range_scale))
+        sample_below_range = -abs(np.random.logistic(0, outside_range_scale))  # type: ignore
```
[the remark this answers is not in the document]
Comment (wbaizer) [reply ergo/metaculus.py:455]: I can't claim that section. Perhaps @ brachbach has an opinion?

[the remark this answers is not in the document]
Comment (wbaizer) [reply ergo/metaculus.py:455]: Honestly I don't remember. I agree that I should have left a comment, but no longer remember what it should say

**Comment by wbaizer** [comment]
stuhlmueller asked me to try out this PR for plotting a prediction I was working on. I noticed some issues while doing so:

1. First I tried just using this PR to make plots. This didn't work:
image
[notebook](https://colab.research.google.com/drive/1tn7LUibCh881BhyzFlcCPXe_ZFwylAsz#scrollTo=a_2oH2vfhzek)

2. Then I tried running the TestVisualPandemic tests. They all failed, some for reasons that are not entirely trivial. I made [this branch](https://github.com/oughtinc/ergo/tree/update_plotting_visual_tests) to try to get them to work, but I couldn't quickly get them to work

3. For some reason `tests/test_metaculus.py::TestMetaculus::test_get_questions_question_status` fails for me locally, though it seems to pass on CI. Maybe run it locally and see if it passes for you, and then if it does I'll just assume that there's something weird with my local setup?

@ djthorne I assume you'll take it from here

**Comment by wbaizer** [comment]
@ djthorne I fixed 3. from above, the testing issue, in https://github.com/oughtinc/ergo/pull/126

(I had written the test incorrectly initially)

**Pull request merged**